zerver/lib/response.py

```diff
@@ -23,16 +24,16 @@ def __init__(self, realm, www_authenticate=None):
 def json_unauthorized(message, www_authenticate=None):
     # type: (text_type, Optional[text_type]) -> HttpResponse
     resp = HttpResponseUnauthorized("zulip", www_authenticate=www_authenticate)
-    resp.content = ujson.dumps({"result": "error",
-                                "msg": message}) + "\n"
+    resp.content = force_bytes(ujson.dumps({"result": "error",
+                                "msg": message}) + "\n")
```

Comment:
Should the lower line be indented to the level of `force_bytes` or `ujson.dumps`?